Source/miniwin/miniwin.h

```diff
@@ -2,8 +2,22 @@
 
 #include <stdint.h>
 
+#include <ctype.h>
+#include <math.h>
+#include "utils/stdarg_compat.h"
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "storm/storm_full.h"

```

Comment:
I don't understand this part. How could moving the files cause you to have to add all these headers?

Reply:
misc.h was combined with miniwin.h. minwin.h used to be a all.h for the minimal implementation of the Windows API used by Diablo, but at this point only misc.h was left and so one was never used without the other and made better sense as a combined file. This also makes it more clear where we need to clean things up in order to get rid of the last parts of miniwin.